vendor/gopkg.in/natefinch/lumberjack.v2/linux_test.go

```diff
@@ -123,7 +123,7 @@ func TestCompressMaintainMode(t *testing.T) {
 	filename2 := backupFile(dir)
 	info, err := os.Stat(filename)
 	isNil(err, t)
-	info2, err := os.Stat(filename2+compressSuffix)
+	info2, err := os.Stat(filename2 + compressSuffix)
```

Comment:
not sure if it's good idea to reformat vendored packages?

Reply:
Sorry. Didn't notice it was in `vendor/`. Reverted.